syntax = "proto3";

message Handshake {
  string userId = 1;
  string key    = 3;
}